wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -1649,6 +1649,14 @@
 "error.conversation.missing_legalhold_consent" = "Due to legal hold, only team members can be added to this conversation";
 "error.message.send.missing_legalhold_consent" = "You cannot send this message because you have at least one outdated device that does not support legal hold. Please update all your devices or remove them from the app settings";
 "error.message.send.title" = "Messages cannot be sent";
+"error.conversation.nfcg.title" = "Group can’t be created";
+"error.conversation.nfcg.message" = "People from backends %s can’t join the same group conversation.
+To create the group, remove affected participants.";
+"error.conversation.nfcg.backend_list_delimeter" = "%s, %s";
+"error.conversation.nfcg.backend_list_end" = "%s and %s";
```

Comment:
Constructing localized strings from separate components is problematic because we're assuming that all languages have the same sentence structure rules as English. But I'm not sure what the alternative solution is here.

Reply:
Yes, it was kind of tricky here. I did not have any other idea for that, because it is not merely singular/plural issue but a collection.